### PR TITLE
amam/fix_localize

### PR DESCRIPTION
**Pull request by mustofa-binary**
# Description

- fix tricky strings cant be translated

Changes:

- use localize inside of the one called the array in help centre and carousel component
- dont go back to homepage after switch language
-- add non greedy regex to match currentpath to the new language path

## Type of change

-   [x] Bug fix
-   [ ] New feature
-   [ ] Breaking change
-   [ ] Dependency update
-   [ ] This change requires a documentation update

### CODE REVIEW

File: src/components/layout/nav.js [hunk 1]
```diff
@@ -199,18 +197,19 @@ const Nav = () => {
...
-                        </NavRightContainer>
+                        <Button onClick={handleLogin} primary>
+                            <span>{localize('Log in')}</span>
+                        </Button>
+                        <SignupButton
+                            secondary
+                            onClick={toggleModal}
+                            move={show_button}
+                            ref={button_ref}
+                            width={button_ref}
```
Comment (oskar-binary) [inline src/components/layout/nav.js]: We only need to pass the ref once and then use the ref inside signupButton to get the width

File: src/components/localization/language-switcher.js [hunk 2]
```diff
@@ -31,12 +31,36 @@ class LanguageSwitch extends Component {
...
+                can be something like /es/about/
+                or just /about/
+                or just /
+            
+            */
+            const current_path = window.location.pathname
+            let destination_path = path
+
+            const current_lang = localStorage.getItem('i18n') || 'en'
+            if (current_lang === 'en') {
```
Comment (oskar-binary) [inline src/components/localization/language-switcher.js]:
```suggestion
            if (value === '/en') {
```

Comment (mustofa-binary) [reply src/components/localization/language-switcher.js]: value is the destination path, meanwhile, we want to know the what is the current url path

Comment (oskar-binary) [reply src/components/localization/language-switcher.js]: line 47 value={`/${this.state.language}`}

**Pull request merged**